_about_livecoms/02_thanks.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: single
+sidebar:
+  nav: thanks.md
+title: Thanks
+excerpt: Thanks for special services in helping make LiveCoMS possible
+permalink: /about/thanks/
+---
+
+LiveCoMS wishes to give special thanks to:
+
+* CU Boulder Libraries, for their generous donation of seed money to start the journal.
+
+* The CU Boulder Department of Chemical and Biological Engineering for
+  agreeing to help serve as publisher, and providing logistical support
+  for journal finances.
+
+* CU Boulder CEAS Dean Bobby Braun and CU Boulder Provost Russell More
+  for their support to have CU Boulder serve as publisher of LiveCoMS.
```

Comment:
Here you also thank CU for publisher role - probably this is the place to have it.

Reply:
Clean up and clarified.